src/components/ViewMilestoneAlerts.jsx

```diff
@@ -12,81 +12,88 @@ import ChangeMilestoneRecipientButton from './ChangeMilestoneRecipientButton';
 import ApproveRejectMilestoneCompletionButtons from './ApproveRejectMilestoneCompletionButtons';
 import WithdrawMilestoneFundsButton from './WithdrawMilestoneFundsButton';
 import DelegateMultipleButton from './DelegateMultipleButton';
+import UserProvider, { Consumer as UserConsumer } from '../contextProviders/UserProvider';
 
 export default function ViewMilestoneAlerts(props) {
   const { currentUser, balance, milestone, campaign } = props;
   const { fullyFunded, status } = milestone;
   const milestoneIsActive = status === 'InProgress' && !fullyFunded;
 
   return (
-    <div>
-      {currentUser && milestoneIsActive && (
-        <ProjectViewActionAlert message="Delegate some donation to this project">
-          <DelegateMultipleButton
-            milestone={milestone}
-            campaign={campaign}
-            balance={balance}
-            currentUser={currentUser}
-          />
-        </ProjectViewActionAlert>
-      )}
+    <UserProvider account={currentUser && currentUser.address}>
```

Comment:
It's wrong way of using provider

Reply:
@aminlatifi Could you link me a good way in our project to I modify this like that?

Reply:
https://github.com/Giveth/giveth-dapp/blob/c5d053b1303750f1fca79ed42d141929341ca84d/src/components/views/Milestones.jsx#L17

Reply:
Above is how the value of UserProvider is consumed.